developer-docs-site/docs/sdks/ts-sdk/typescript-sdk.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Typescript SDK"
+slug: "typescript-sdk"
+---
+
+# Typescript SDK
+
+Aptos provides an official Typescript SDK. This section describes how to install and use it.
+
+## Installing Typescript SDK
+
+1. Make sure you [downloaded the latest precompiled binary for the Aptos CLI](/cli-tools/aptos-cli-tool/install-aptos-cli/#download-precompiled-binary).
```

Comment:
Why does this relate to CLI?

Reply:
The npmjs version SDK is recommended with the pre-built binary of CLI. It is not needed, but we need to enforce this pairing, until things change.